lib/dotcom/system_status/alerts.ex

```diff
@@ -0,0 +1,103 @@
+defmodule Dotcom.SystemStatus.Alerts do
+  @moduledoc """
+  A utility module intended to filter alerts for the system status feature,
+  relying on some specific criteria that are specific enough that they don't
+  belong in the main `Alerts` module.
+  """
+  @relevant_effects [:delay, :shuttle, :suspension, :station_closure]
+
+  defp has_started?(active_period_start, now) do
```

Comment:
Same thing with all of these. `now` should be a more generic `date_time` or something.

Reply:
[Done!](https://github.com/mbta/dotcom/pull/2327/commits/2f0f0ec3bb88ec1ad83a72a5ecfe8e70159a22c0)